www/js/app.js

```diff
@@ -906,14 +906,15 @@ define(['jquery', 'zimArchiveLoader', 'util', 'uiUtil', 'cookies','abstractFiles
                     // The content is fully loaded by the browser : we can hide the spinner
                     $("#searchingArticles").hide();
                     // Deflect drag-and-drop of ZIM file on the iframe to Config
-                    var doc = iframeArticleContent.contentDocument.documentElement;
+                    var doc = iframeArticleContent.contentDocument;
+                    doc = doc ? doc.documentElement : null;
```

Comment:
It would be better to use different variable names here for the contentDocument and its documentElement to avoid confusion